src/main/java/com/google/firebase/auth/FirebaseAuth.java

```diff
@@ -841,6 +919,65 @@ protected Void execute() throws FirebaseAuthException {
     };
   }
 
+  /**
+   * Deletes the users specified by the given identifiers.
+   *
+   * <p>Deleting a non-existing user won't generate an error. (i.e. this method is idempotent.)
```

Comment:
Suggest:

"Deleting a non-existing user does not generate an error (the method is idempotent). Non-existing users are considered to be successfully deleted and are therefore included in the `DeleteUsersResult.getSuccessCount()` value."

Reply:
Done.